website/static/js/jquery.homeycombs.js

```diff
@@ -0,0 +1,145 @@
+(function($) {

```

Comment:
That is true. On the other hand, for files that are loaded for every page (the basic jquery, for example), CDNs are optimized to serve them. They are geographically distributed, so it can lower the burden on our server (even if, for now, I do not expect the load on the server to be that gigantic). Also, the headers of the CDNs are optimized for caching purposes, so it helps reducing the number of fetches.